app/src/main/java/org/openedx/app/AppActivity.kt

```diff
@@ -134,6 +138,43 @@ class AppActivity : AppCompatActivity(), InsetHolder, WindowSizeHolder {
         }
     }
 
+    override fun onStart() {
+        super.onStart()
+
+        if (viewModel.isBranchEnabled) {
+            val callback = BranchUniversalReferralInitListener { _, linkProperties, error ->
+                if (linkProperties != null) {
+                    Log.i(BRANCH_TAG, "Branch init complete.")
```

Comment:
Can you please use logger from `org.openedx.core.utils`. If there is not enough log levels there, feel free to add them :)